ContactDetailsApi.Tests/V1/E2ETests/Fixtures/ContactDetailsFixture.cs

```diff
@@ -54,12 +58,13 @@ protected virtual void Dispose(bool disposing)
         private IEnumerable<ContactDetailsEntity> CreateContacts(int count, bool isActive)
         {
             return _fixture.Build<ContactDetailsEntity>()
-                .With(x => x.CreatedBy, () => _fixture.Build<CreatedBy>()
-                    .Create())
+                .With(x => x.CreatedBy, () => _fixture.Build<CreatedBy>().Create())
                 .With(x => x.RecordValidUntil, DateTime.UtcNow)
                 .With(x => x.IsActive, isActive)
                 .With(x => x.TargetType, TargetType.person)
-                .With(x => x.TargetId, TargetId).CreateMany(count);
+                .With(x => x.TargetId, TargetId)
+                .Without(x => x.VersionNumber)
```

Comment:
I've previously added version number like this ` .With(x => x.VersionNumber, (int?) null)`, not a major different but yeah

Reply:
Why do you think that is better?

Reply:
I think Callum's version is a lot clearer. explicit cast to nullable int...? hmmm.....